addon/components/organism/whats-on/schedule/template.hbs

```diff
@@ -0,0 +1,17 @@
+<div class='whats-on-schedule-metadata'>
+    <div class='whats-on-schedule-metadata-image'>
+        <img
+            alt="image for {{airing.showTitle}}"
+            src={{airing.show.image.url}}>
+    </div>
+    <div class="whats-on-schedule-metadata-text-outer">
```

Comment:
why is this text-outer? where's the inner?  am i missing something?

Reply:
renamed these classes (used `copy` as in its `written matter` context)

Reply:
also broke the `schedule` component styles out into their own file and renamed the classes accordingly, in the same manner as `episode`.